### PR TITLE
chore: add documentation issue template

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [X] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3-documentation.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      label: Which browsers have you used?
+      description: You may select more than one.
+      options:
+        - label: Chrome
+        - label: Firefox
+        - label: Safari
+        - label: Edge
+
+  - type: checkboxes
+    id: operating-systems
```
Comment (vladimirpotekhin) [inline .github/ISSUE_TEMPLATE/3-documentation.yml:35]: nit: I'm not sure we need information about operating system in this type of bug-report 🤔

**Review by vladimirpotekhin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/3-documentation.yml (hunk 1 above)
Comment (splincode) [reply .github/ISSUE_TEMPLATE/3-documentation.yml:35]: `iOS/Android`

Comment (nsbarsukov) [reply .github/ISSUE_TEMPLATE/3-documentation.yml]: > `iOS/Android`

@splincode do you think that it is better to keep this section ?

Comment (splincode) [reply .github/ISSUE_TEMPLATE/3-documentation.yml]: I think so, but as you wish

Comment (nsbarsukov) [reply .github/ISSUE_TEMPLATE/3-documentation.yml:35]: I keep it (to be similar to Taiga UI)

**Pull request merged**